### PR TITLE
Add error handling component for wrong url

### PR DESCRIPTION
**Pull request by mayakappen**
-Adds a new component for error handling (PageNotFound)
-Component is last Route in the Switch
-if no other Routes match then the PageNotFound component is rendered
-User is able to navigate back home from PageNotFound
*It says 404 page not found right now but we could change the message if we need to

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import Navbar from '../Navbar/Navbar';
 import Movie from '../Movie/Movie';
 import AllMovies from '../All-Movies/All-Movies';
+import PageNotFound from '../PageNotFound'
```
Comment (mayakappen) [inline src/App/App.js:5]: I only made one file, no scss or anything

File: src/App/App.js [hunk 2]
```diff
@@ -59,6 +60,7 @@ class App extends Component {
         <Switch>
           <Route exact path='/' render={() => <AllMovies movies={this.state.movies}/>}/>
           <Route exact path='/:id' render={({ match }) => <Movie id={match.params.id}/>}/>
+          <Route component={PageNotFound} />
```
Comment (mayakappen) [inline src/App/App.js:63]: This I believe is all we need to cover if any other route is entered

File: src/PageNotFound.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './styles.scss'
+
+const PageNotFound = () => {
+    return (
+        <div style={{display: "flex",
```
Comment (mayakappen) [inline src/PageNotFound.js:6]: Inline style because its an easy layout

**Review by mayakappen — COMMENTED** [review]
Just a simple component that gets displayed if any non matching url is entered

**Review by aspitz1 — APPROVED** [review]
I love the simplicity of this! It'll add an extra safety net. Thank you!!

**Pull request merged**